config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root "users#index"
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :create, :show] do
```

Comment:
- [ ] I can get a form for adding a new post but upon submitting I am getting an error please make sure the form lead to the correct route

Reply:
![Uploading Screenshot 2023-11-03 at 16.26.22 1.png…]()